app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, foreign_key: 'User'
```

Comment:
- [ ] A through association should have the alias model **refer** to actual model (_**User**_) using `class_name`, instead of `foreign_key`. As per the given _ERD_, the _Like_ **association** should have _Like_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 
   
   Kindly ensure to improve upon this aspect.